jinja2/base.html

```diff
@@ -211,34 +244,83 @@
   </div></main>
 
   <!-- Footer -->
-  <footer id="main-footer"><div class="center">
-    <div class="column-container">
-      <div class="column-main">
-        {% block footer_copyright %}
+  {% if waffle.flag('redesign') %}
+      <footer id="nav-footer" class="nav-footer"><div class="center">
```

Comment:
Nit: `footer` and open `div` on same line. Should probably break the `div` to the next line.